styx-flyte-client/src/main/java/com/spotify/styx/flyte/client/FlyteInputsUtils.java

```diff
@@ -56,7 +56,16 @@ private FlyteInputsUtils() {
         .forEach(
             (key, value) -> {
               if (!key.toLowerCase().equals(PARAMETER_NAME)) {
-                literalMapBuilder.putLiterals(key, value.getDefault());
+                if (value.hasDefault()) {
```

Comment:
have we verified that this will be populated in the LP?

Reply:
Yes I believe so. 
`
* @return Whether the default field is set.
*/
public boolean hasDefault() {
    return behaviorCase_ == 2;
}
`